featuretests/upgrade_test.go

```diff
@@ -196,6 +198,25 @@ func (s *upgradeSuite) TestDowngradeOnMasterWhenOtherControllerDoesntStartUpgrad
 	}
 	s.PatchValue(&upgradesteps.IsMachineMaster, fakeIsMachineMaster)
 
+	vers := version.Binary{
```

Comment:
I'm unclear why we need this block of code. Can you add a comment?
We're not changing the host series so why do we need the extra agent binary tarball written?